example/package.json

```diff
@@ -10,12 +10,13 @@
     "precommit": "npm run lint",
     "snapshot": "fedx-scripts jest --updateSnapshot",
     "start": "fedx-scripts webpack-dev-server --progress",
+    "start:with-theme": "paragon install-theme && npm start",
```

Comment:
When running this, I get the following error:

```
❯ npm run start:with-theme

> example@1.0.0 start:with-theme
> paragon install-theme && npm start

sh: paragon: command not found
npm ERR! Lifecycle script `start:with-theme` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: example@1.0.0 
npm ERR!   at location: /Users/astankiewicz/Desktop/edx/paragon/example 
```

I believe we could resolve this error by changing the line to:

```
../bin/paragon-scripts.js install-theme && npm start
```